#7/README.md

```diff
@@ -0,0 +1,28 @@
+# Proof-of-Membership
+
+## Former Participation in Celo Hackathons
+
+- EthGlobal Istanbul
+
+## Goal for this Proof of Ship
+
+What are we planning to achieve. Be as detailed as possible. Try to break it down into weekly milestones.
```

Comment:
Please provide the goals, that you are planning to achieve this month.

Reply:
Thanks, I will add the goals planned for this month and fix the directory name to put it in 1 instead of 7 because I was confused with the Pull Request numbers.

Reply:
sounds good. I realized the instructions/ guides were not clear enough.

Reply:
And having # in directory name makes it difficult in Linux terminal, so would you mind removing it to keep just the number?

Reply:
done